static/js/overlays.js

```diff
@@ -42,6 +42,19 @@ exports.recent_topics_open = function () {
     return open_overlay_name === "recent_topics";
 };
 
+exports.is_flatpickr_open = function () {
+    return $(".custom_user_field_value").hasClass("active");
+};
+
+exports.hide_flatpickr = function () {
+    const element = $(".flatpickr-calendar");
+    if (element.hasClass("open")) {
+        element.removeClass("open");
+    }
+    $("#account-settings").trigger("click");
```

Comment:
If this is for focus, `trigger("focus")` would be more direct.

Reply:
This is what baffles me as well , <code>trigger("focus")</code> doesn't relaunch the <code>faltpickr</code> once the user presses the <code>Esc</code> key . However <code>trigger("click")</code> does the job .